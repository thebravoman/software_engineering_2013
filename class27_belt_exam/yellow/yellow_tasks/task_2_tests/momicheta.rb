require "csv"
godini_sort=Array.new
relationship = ARGV[1]
CSV.foreach(ARGV[0]) do |row|
	if row[3]== relationship
		godini_sort << row
		p row
	end
end

godini_sort.sort_by! {|e| e[1].to_i}

CSV.open("momicheta_result.csv","w") do |csv|
	godini_sort.each do |e|
		csv << e
	end
end 