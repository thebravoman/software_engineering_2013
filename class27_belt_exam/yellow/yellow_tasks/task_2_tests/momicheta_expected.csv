Kremena,11,Drujba,Sama,0897425268
Ivana,12,Liulin,Sama,0886712324
Iva,13,Lozenec,Sama,0884712522
Ivana,15,Liulin,Sama,0886734224
Dea,15,Goce Delchev,Sama,076418795
Ivana,16,Liulin,Sama,0886712224
Ivana,19,Liulin,Sama,0886712544
Mariq,21,Iztok,Sama,0876125485
Kremena,21,Drujba,Sama,0897451268
Dea,21,Goce Delchev,Sama,076418435
Doroteq,22,Goce Delchev,Sama,0887491245
Mariq,23,Iztok,Sama,0876123485
Iva,23,Lozenec,Sama,0884712512
Ivet,23,Goce Delchev,Sama,088663558
Ivet,24,Goce Delchev,Sama,0886413558
Ivet,24,Goce Delchev,Sama,0886412558
Ivet,25,Goce Delchev,Sama,0882313558
Mariq,26,Iztok,Sama,0876123443
Doroteq,28,Goce Delchev,Sama,0893491245
Mariq,29,Iztok,Sama,0876123409
Iva,33,Lozenec,Sama,0884712564
Dea,35,Goce Delchev,Sama,076418764
Kremena,41,Drujba,Sama,0897463268
Doroteq,52,Goce Delchev,Sama,08874923245
Doroteq,52,Goce Delchev,Sama,0888591245
Kremena,71,Drujba,Sama,0897444268
Iva,73,Lozenec,Sama,0884712574
Dea,75,Goce Delchev,Sama,076413695
